scripts/image-diff

```diff
@@ -0,0 +1,81 @@
+#!/usr/bin/env python
```

Comment:
Please add a license identifier. Examples can be found in the other scripts, but it is generally just `SPDX-License-Identifier: BSD-3-Clause` to declare the file as using the Freedoom license.